aexpect/remote_door.py

```diff
@@ -438,7 +455,7 @@ def set_subcontrol_parameter_object(subcontrol, value):
 
     LOG.info("Sharing the test parameters over the network")
     Pyro4.config.AUTOPROXY = False
-    Pyro4.config.REQUIRE_EXPOSE = False
+    expose_remote_classes(["virttest.utils_params.Params"])
```

Comment:
I don't think this belongs here. The aexpect project is generic, not tied to avocado-vt/virttest.